python/tvm/relay/frontend/onnx.py

```diff
@@ -2254,8 +2255,13 @@ def from_onnx(model,
 
     Parameters
     ----------
-    model : protobuf object
-        ONNX ModelProto after ONNX v1.1.0
+    model_path : str
+        The path of the ONNX model
```

Comment:
I recommend allowing the `model` argument to be either an Onnx ModelProto or a string pointing to the saved Onnx file. You can then add a check to see if its a string and do `onnx.load` and `load_external_data_for_model`. Otherwise parse the OnnxProto as normal. This will make it a lot easier to deal with the existing TVM codebase and allow your tests to pass.